README.md

```diff
@@ -148,7 +148,7 @@ Users acknowledge that no warranty is being made of a successful installation. S
 | Erigon     | Prysm      | Prysm      |
 | Besu       | Teku       | Teku       |
 
-### Goerli
+### Goerli (Unsupported)
```

Comment:
Please remove this section, and everything that mentions Goerli in the readme and the docs. Unless is to say that it's unsupported